src/codegen/general.rs

```diff
@@ -514,16 +514,13 @@ pub fn declare_default_from_new(
 /// Escapes string in format suitable for placing inside double quotes.
 pub fn escape_string(s: &str) -> String {
     let mut es = String::with_capacity(s.len() * 2);
-    let _ = s
-        .chars()
-        .map(|c| match c {
-            '\"' | '\\' => {
-                es.push('\\');
-                es.push(c)
-            }
-            _ => es.push(c),
-        })
-        .count();
+    for c in s.chars() {
+        match c {
+            '\"' | '\\' => es.push('\\'),
+            _ => (),
+        }
+        es.push(c)
```

Comment:
nightly clippy strart nags about this and other `.count()`

Reply:
What exactly does it complain about, can you paste?

Reply:
```
warning: this call to `map()` won't have an effect on the call to `count()`
   --> src\codegen\general.rs:517:13
    |
517 |       let _ = s
    |  _____________^
518 | |         .chars()
519 | |         .map(|c| match c {
520 | |             '\"' | '\\' => {
...   |
525 | |         })
526 | |         .count();
    | |________________^
    |
    = note: `#[warn(clippy::suspicious_map)]` on by default
    = help: make sure you did not confuse `map` with `filter`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_map
```

Reply:
Maybe you want `for_each()` here instead?

Reply:
IMHO just `for` is better, no closure created etc.

Reply:
For this kind of code, I agree, yes.

The closure is optimized away btw, but with a for loop it's simply more readable IMHO :)